api/v1alpha1/ext_authz_type.go

```diff
@@ -0,0 +1,12 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+// ExtAuthz defines the configuration for External Authorization.
```

Comment:
should this be called `extAuthz` or `externalAuth` ?

Reply:
I named after the envoy filter ` envoy.filters.network.ext_authz`